src/Compilers/Core/Portable/Symbols/ITupleTypeSymbol.cs

```diff
@@ -14,7 +13,7 @@ namespace Microsoft.CodeAnalysis
     /// </remarks>
     public interface ITupleTypeSymbol : ITypeSymbol
```

Comment:
We should simply extend INamedTypeSymbol for this. For example we didn't add new interface for anonymous types.